config/kubernetes/sig-release/teams.yaml

```diff
@@ -86,6 +86,7 @@ teams:
     - lachie83 # PM
     - lavalamp # API Machinery
     - liggitt # Auth / Release
+    - lingxiankong # Cloud Provider / OpenStack
```

Comment:
```suggestion
    - lingxiankong # OpenStack
```

Reply:
Making a code suggestion here as IIUC @lingxiankong will be a milestone maintainer for OpenStack, but not top-level SIG Cloud Provider.

Reply:
Done